plugin/src/main/resources/data/passives.yml

```
@@ -0,0 +1,19 @@
+  - id: regeneration
+    userFacing: false
+
+  - id: arrow_recharge
+    displayItem: bundle
+    userFacing: true
+    metadata:
+      arrowIntervalTicks: 20
+
+  - id: barrage
+    displayItem: spectral_arrow
+    userFacing: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)